python/xorbits/_mars/tensor/fuse/jax.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2022-2023 XProbe Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+try:
+    import jax
+
+    JAX_INSTALLED = True
+except ImportError:
+    JAX_INSTALLED = False
+
+from .. import arithmetic, reduction
+from ..array_utils import as_same_device
+from ..operands import TensorFuse
+from .core import TensorFuseChunkMixin
+
+
+class TensorJAXFuseChunk(TensorFuse, TensorFuseChunkMixin):
```

Comment:
I think we need to figure out a way to test this class. Do you have any idea?

Reply:
Yeah I agree. I was thinking about disabling the numexpr engine during the jax workflow, so only jax and cupy will be used for optimization and we can thus test jax, but i haven't found an easy way to do that 🤔

Reply:
Placen jax in front of numexpr in `engines`, and jax will be chosen firstly if it is installed. Numexpr is also be tested when jax is not installed.